models/classes/class.QtiTestCompiler.php

```diff
@@ -476,7 +476,7 @@ protected function compactTest()
         $resolver = new taoQtiTest_helpers_ItemResolver(Service::singleton());
         $originalDoc = $testService->getDoc($test);
 
-        $compiledDoc = XmlCompactDocument::createFromXmlAssessmentTestDocument($originalDoc, $resolver, $resolver);
+        $compiledDoc = XmlCompactDocument::createFromXmlAssessmentTestDocument($originalDoc, $resolver, $resolver, '2.2');
```

Comment:
i would go for a constant that will describe the new added value

Reply:
Added a constant.